### ISSUE TITLE
[REVIEW] aparna2071

### ISSUE THREAD

**Issue by aparna2071** [EddieHubCommunity/support#3865]
### GitHub Profile Link

https://github.com/aparna2071

**Comment by shubhimainalli** [comment]
Hey Aparna, your portfolio looks awesome!!
Small suggestion: instead of writing "How to reach me" you can make it as reachout to me or contact me.

**Comment by aparna2071** [comment]
Thank you for your suggestion, @shubhimainalli. I will make the modifications you mentioned.

**Comment by schmelto** [comment]
> Hey Aparna, your portfolio looks awesome!! Small suggestion: instead of writing "How to reach me" you can make it as reachout to me or contact me.

Yes I thin a "call to action" is much nicer

**Comment by aparna2071** [comment]
Thanks @schmelto for your suggestion. You mean instead of "How to reach me", I should write "call to action" , right?

**Comment by schmelto** [comment]
> Thanks @schmelto for your suggestion. You mean instead of "How to reach me", I should write "call to action" , right?

No. I mean a call to action like "Connect with me:" is better than "How to reach me" but I think you already implemented that.

**Comment by aparna2071** [comment]
Ok ok @schmelto I got it, yes I had implemented it.
Thanks
